pkgs/development/tools/rust/cargo-llvm-cov/default.nix

```
@@ -0,0 +1,36 @@
+{ lib, rustPlatform, fetchCrate, fetchFromGitHub, libllvm, makeBinaryWrapper }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-llvm-cov";
+  version = "0.4.14";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-DY5eBSx/PSmKaG7I6scDEbyZQ5hknA/pfl0KjTNqZlo=";
+  };
+
+  postPatch = ''
+    rm tests/test.rs
+  '';
+
+  nativeBuildInputs = [ makeBinaryWrapper ];
```

`makeWrapper` is much more commonly used over `makeBinaryWrapper` for simple wrapping like this:
```suggestion
  nativeBuildInputs = [ makeWrapper ];
```

---

Guess `makeBinaryWrapper` is preferred, per manual. Wonder why we don't use it more...